resolwe_bio/docker_images/dnaseq/packages-manual/r-packages.sh

```diff
@@ -2,18 +2,21 @@
 
 . /var/cache/build/packages-manual/common.sh
 
-# These steps need to be performed because certain packages produce a "database corrupt error". The solution is to
-# restart R, but this is not feasible during the docker built or building of dependencies. This chunk also installs
-# packages from bioconductor. This entire step should be considered temporary until we can get packrat to work for
-# packages from all universes (CRAN, Bioconductor).
-Rscript --slave --no-save --no-restore-history -e "install.packages('assertthat')"
+# Pin BiocManager version 3.13
+Rscript --slave --no-save --no-restore-history -e "install.packages('BiocManager')"
+Rscript --slave --no-save --no-restore-history -e "BiocManager::install(version = '3.13', ask = FALSE)"
```

Comment:
Pin bioconductor version using https://rstudio.github.io/renv/reference/settings.html#settings and forego the installation of the whole bioconductor universe.